src/CompileErrorProcessor.ts

```diff
@@ -79,13 +76,31 @@ export class CompileErrorProcessor {
         }
     }
 
+    public sendErrors(): Promise<void> {
+        //session is shuttind down, process logs immediately
```

Comment:
spelling mistake `shuttind`. Also, `process logs immediately`...where is that happening?